babyagi.py

```diff
@@ -388,7 +421,8 @@ def main ():
 
             result_id = f"result_{task['task_id']}"
 
-            results_storage.add(task, result, result_id, vector)
+            if USE_PINECONE:
```

Comment:
Why is this? The results storage shouldn't be affected by the manual chat. Also this will affect Chroma users as well.

Reply:
There is OpenAi call to get embeddings there. If you dont have an api key this will not get through. :)